csharp/ql/lib/semmle/code/csharp/frameworks/Format.qll

```diff
@@ -8,67 +8,91 @@ private import semmle.code.csharp.frameworks.System
 private import semmle.code.csharp.frameworks.system.Text
 
 /** A method that formats a string, for example `string.Format()`. */
-class FormatMethod extends Method {
-  FormatMethod() {
-    exists(Class declType | declType = this.getDeclaringType() |
+abstract class FormatMethod extends Method {
```

Comment:
Now we are exposing an abstract method, which is not backwards-compatible (and also not best practice). So Instead replace the current abstract `FormatMethod` class with a private `FormatMethodImpl` class, and add `final class FormatMethod = FormatMethodImpl;`.

Reply:
Never mind, I see that you did exactly this in a subsequent commit.